styles/twitch/catppuccin.user.css

```diff
@@ -625,6 +625,17 @@
       }
     }
 
+    /* VOD chapters */
+    .preview-card-game-balloon__content .tw-interactable:hover {
+      p,
+      figure {
+        color: @text !important;
+      }
+      .media-row__image-play-overlay {
+        background-color: fade(@mantle, 80%) !important;
```

Comment:
Hmm fading mantle? This should just be base/surface0/surface1 then since thats the shade you'll end up with I'd think.

Reply:
No I mean if you are fading a shade, like a non-accent color (crust, mantle, base, surface0, ..., subtext1, text) you should just use a different shade (unless you are fading it to a very low opacity for a different reason, such as an overlay on top of a video thumbnail).

Reply:
So instead of `fade(@mantle, 80%)` or `fade(@base, 80%)`, use `@surface0`.

Reply:
The background should be transparent, it is overlapping category cover art:
![2024-11-12-164934_hyprshot](https://github.com/user-attachments/assets/2eb6567c-d105-4950-8c73-9788570abd11)

Reply:
Oh gotcha so it isn't just behind the cover art. Sure, you can fade it. Go back to mantle if you want if you think that suits it better.

Reply:
Actually, I think that the base color looks better.